config/settings_local.py

```diff
@@ -7,25 +7,29 @@
 SECRET_KEY = get_random_secret_key()  # pragma: allowlist secret
 
 # Minimal group for read access of public flaws in OSIDB
-PUBLIC_READ_GROUPS = ["data-prodsec"]
+PUBLIC_READ_GROUPS = [
+    "data-prodsec",
+    "osidb-stage-public-read",
+]
 # Minimal group for write access of public flaws in OSIDB
-PUBLIC_WRITE_GROUP = "data-prodsec-write"
+PUBLIC_WRITE_GROUP = ["data-prodsec-write", "osidb-stage-public-write"]
 # Minimal group for read access of embargoed flaws in OSIDB
-EMBARGO_READ_GROUP = "data-topsecret"
+EMBARGO_READ_GROUP = ["data-topsecret", "osidb-stage-embargo-read"]
 # Minimal group for write access of embargoed flaws in OSIDB
-EMBARGO_WRITE_GROUP = "data-topsecret-write"
+EMBARGO_WRITE_GROUP = ["data-topsecret-write", "osidb-stage-embargo-write"]
 # Minimal group for read access of internal flaws in OSIDB
-INTERNAL_READ_GROUP = "data-internal-read"
+INTERNAL_READ_GROUP = ["data-internal-read", "osidb-stage-internal-read"]
 # Minimal group for write access of internal flaws in OSIDB
-INTERNAL_WRITE_GROUP = "data-internal-write"
+INTERNAL_WRITE_GROUP = ["data-internal-write", "osidb-stage-internal-write"]
```

Comment:
The groups from the settings are used throughout the code and wherever there is a singular really just one group is expected so I think this will break the local environment on number of places.

Reply:
ok if that is the design then I can change this the other way around eg. just rewrite everything to use local group (and not add any groups here.

Reply:
I think it is a bad design somehow organically achieved. But it is as it is now. We can fix the design which may be more costly or you can implement your change some other way

Reply:
as discussed in chat - for now we will just rewrite acls and not add any new groups here and raise an issue about future redesign in this area.